### PR TITLE
add bintray upload support

### PR DESCRIPTION
**Pull request by OrenZak**
Add a modular Bintray distribution functionality so we can use in other repos containing lib modules.

There are 2 files that can be shared among other repos.
1. `gradle/release_bintray.gradle`
2. `gradle/release_maven.gradle`
Those two files are creating the task to publish the modules.
* `release.gradle` -> should be copied and set the relevant values inside.
* The modules should follow the guideline I've set, so each module should have a `gradle.properties` file with the content as you can see in: `ui/gradle.properties`.
* Add the values as you can see in `gradle.properties` on the main directory.
* Create `bintray.properties` on the main directory with credentials for bintray.

* After is all set run:
   * ```gradle install```
   * ```gradle bintrayUpload```

cc: @yosriz @drEfrati

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -2,11 +2,17 @@
...
 buildscript {
     ext {
+        // Dependencies
         kotlin_version = '1.2.71'
         support_version = '26.1.0'
         constraint_layout_version = '1.1.3'
         kin_ecosystem_base_version = '1.1.1'
 
+        // Plugins
+        bintrayVersion = '1.8.4'
```
Comment (yosriz) [inline build.gradle:12]: can it become common import somehow?
i.e. no need to copy it to each project

File: build.gradle [hunk 2]
```diff
@@ -15,12 +21,13 @@ buildscript {
     dependencies {
         classpath 'com.android.tools.build:gradle:3.2.1'
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
-        // NOTE: Do not place your application dependencies here; they belong
-        // in the individual module build.gradle files
+        classpath "com.jfrog.bintray.gradle:gradle-bintray-plugin:$bintrayVersion"
```
Comment (yosriz) [inline build.gradle:24]: can it become common import somehow?
i.e. no need to copy it to each project

**Review by yosriz — APPROVED** [review]
@OrenZak not match I can observer here,
can you elaborate what should be shared code? (or extract it?)
also an example of a published AAR.

File: build.gradle (hunk 2 above)
Comment (OrenZak) [reply build.gradle:24]: So for dependencies, each project should specify it's own(also for the above file).

**Pull request merged**